vendor/bat-native-ledger/src/bat_get_media.cc

```diff
@@ -928,21 +928,54 @@ std::string BatGetMedia::getYoutubeMediaKeyFromUrl(
   return std::string();
 }
 
+// static
 std::string BatGetMedia::getYoutubePublisherKeyFromUrl(const ledger::VisitData& visit_data) {
```

Comment:
I don't see anything else in visit_data being used other than path. Can we change this to just accept `const std::string& path`?

Reply:
done